Lib/datetime.py

```diff
@@ -1033,10 +1033,23 @@ def ctime(self):
             self._day, self._year)
 
     def strftime(self, fmt):
-        """
-        Format using strftime().
-
-        Example: "%d/%m/%Y, %H:%M:%S"
+        """Convert to a string in the given format via time.strftime().
```

Comment:
I think these aren't supposed to be `time.strftime()`, I think we want to directly reference `` :manpage:`strftime(3)` ``. Or at least `strftime(3)` (I guess these docstrings aren't destined for Sphinx, so probably just `strftime(3)` is most appropriate)